WordPress/Classes/ViewRelated/Plans/PlanDetailViewController.swift

```diff
@@ -96,6 +96,8 @@ class PlanDetailViewController: UIViewController {
         planTitleLabel.text = plan.fullTitle
         planDescriptionLabel.text = plan.description
         planPriceLabel.text = priceDescriptionForPlan(plan)
+
+        purchaseButton.alpha = (plan == .Free) ? 0 : 1
```

Comment:
I've updated the code to switch out the button based on whether the plan is free / active.